manifests/charts/gateway/templates/deployment.yaml

```diff
@@ -21,6 +21,9 @@ spec:
         {{- toYaml . | nindent 8 }}
       {{- end }}
       labels:
+        {{- with .Values.labels }}
```

Comment:
is this going to have duplicates with L31?

Reply:
Yes, it's included in .Values.labels. Thanks for the reminder. Fixed.